corgi/api/filters.py

```diff
@@ -62,10 +62,10 @@ class Meta:
 
     # User gave a filter like ?ofuri= in URL, assume they wanted a stream
     ofuri = CharFilter(field_name="productstreams", lookup_expr="ofuri")
-    products = CharFilter(lookup_expr="ofuri")
-    product_versions = CharFilter(field_name="productversions", lookup_expr="ofuri")
-    product_streams = CharFilter(field_name="productstreams", lookup_expr="ofuri")
-    product_variants = CharFilter(field_name="productvariants", lookup_expr="ofuri")
+    products = CharFilter(lookup_expr="name")
```

Comment:
If Hardik really does want a substring / pattern match, where searching for "rhel" shows both "rhel" and "rhel-br" too, switch this code to below:
```suggestion
    products = CharFilter(lookup_expr="name__icontains")
```